src/ResultView.tsx

```diff
@@ -43,16 +72,12 @@ const ResultView = ({ result }: { result: any }) => {
           </div>
           <ul className="sui-result__details">
             <li>
-              <span className="sui-result__label">Industries: </span>
-              <span
-                className="sui-result__value"
-                dangerouslySetInnerHTML={{
-                  __html:
-                    industries && industries.raw
-                      ? industries.raw.join(", ")
-                      : "NA",
-                }}
-              />
+              {industries.raw.map((industry: any) => (
+                <Chip
+                  style={{ marginRight: 10, marginBottom: 7 }}
+                  label={industry}
+                />
+              ))}
```

Comment:
Looks like your branch is outdated, this shouldn't be changed since we had already merged in your chips in the past.